docs/mosip-installation-without-public-dns.md

```diff
@@ -46,48 +53,90 @@
+## Update config properties below starting deployment
+* Add/Update the below property in `application-default.properties` and comment on the below property in the `*-default.properties` file in the config repo.
   ```
   mosip.iam.certs_endpoint=http://${keycloak.external.host}/auth/realms/mosip/protocol/openid-connect/certs
   ```
-* Restart admin, resident, and partner services.
+* Add/Update the below property in the `esignet-default.properties` file in the config repo.
+  ```
+  spring.security.oauth2.resourceserver.jwt.jwk-set-uri=http://${keycloak.external.host}/auth/realms/mosip/protocol/openid-connect/certs
+  ```
 
-## Local setup
+## Steps to follow while deployment
+* While install few modules, installation script prompts if you have public domain and valid ssl certificates on the server.
+  Opt option `n` as we are using self-singed certificates.
+  For example:
+  ```
+  $ ./install.sh
+  Do you have public domain & valid SSL? (Y/n) 
+   Y: if you have public domain & valid ssl certificate
+   n: If you don't have a public domain and a valid SSL certificate. Note: It is recommended to use this option only in development environments.
+  ```
 
+## Local/Client side setup
 ### DNS setup
 * Map IP address and domain names to set up DNS in the respective `hosts` file. <br>
   For example: `/etc/hosts` files for Linux machines.
   ```
-   <PUBLIC_IP>    api.sandbox.xyz.net resident.sandbox.xyz.net idp.sandbox.xyz.net prereg.sandbox.xyz.net
-   <INTERNAL_IP>  sandbox.xyz.net api-internal.sandbox.xyz.net activemq.sandbox.xyz.net kibana.sandbox.xyz.net regclient.sandbox.xyz.net admin.sandbox.xyz.net minio.sandbox.xyz.net iam.sandbox.xyz.net kafka.sandbox.xyz.net postgres.sandbox.xyz.net pmp.sandbox.xyz.net onboarder.sandbox.xyz.net smtp.sandbox.xyz.net
+   <PUBLIC_IP>    api.sandbox.xyz.net resident.sandbox.xyz.net esignet.sandbox.xyz.net prereg.sandbox.xyz.net healthservices.sandbox.xyz.net
+   <INTERNAL_IP>  sandbox.xyz.net api-internal.sandbox.xyz.net activemq.sandbox.xyz.net kibana.sandbox.xyz.net regclient.sandbox.xyz.net admin.sandbox.xyz.net minio.sandbox.xyz.net iam.sandbox.xyz.net kafka.sandbox.xyz.net postgres.sandbox.xyz.net pmp.sandbox.xyz.net onboarder.sandbox.xyz.net smtp.sandbox.xyz.net compliance.sandbox.xyz.net
   ```
-* Visit `https://sandbox.xyz.net/` on any browser to test DNS working.
-
-### Browser setup
-* Delete `hsts` security policies for listed domains.
-    * open `hsts` setting in any browser example: `chrome://net-internals/#hsts`.
-    * Provide domain name `iam.sandbox.xyz.net`and click on `delete`.
-      ![mosip-without-dns-2.png](_images/mosip-without-dns-2.png)
-    * Repeat the same for other domains.
+* Visit https://sandbox.xyz.net on any browser. You should see the landing page.
+  In case you are faced with a certificate exception, accept the certificate and proceed because we are using the self-signed certificate.
```

Comment:
In case you face a certificate exception, accept the certificate and proceed because we are using the self-signed certificate.